contracts/HubPool.sol

```diff
@@ -74,9 +74,26 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     // Whether the bundle proposal process is paused.
     bool public paused;
 
-    // Whitelist of origin token + ID to destination token routings to be used by off-chain agents. The notion of a
-    // route does not need to include L1; it can be L2->L2 route. i.e USDC on Arbitrum -> USDC on Optimism as a "route".
-    mapping(bytes32 => address) private whitelistedRoutes;
+    // A Route is mapped to a hash of an origin chain ID, origin token address, and destination chain ID. Those three
```

Comment:
Is this meant to fix an audit issue? Could you put the text of the issue in the PR description and label the issue in the title, ex: `refactor: [M04] ...`?

Reply:
Done